content/docs/command-reference/exp/run.md

```diff
@@ -1,8 +1,15 @@
 # exp run
 
-Run or resume a [DVC experiment].
+Run or resume a [DVC experiment] based on a [DVC pipeline].
 
 [dvc experiment]: /doc/user-guide/experiment-management/experiments-overview
+[dvc pipeline]: /doc/user-guide/pipelines
+
+<admon type="info">
+
+This is equivalent to `dvc repro` followed by `dvc exp save`.
```

Comment:
For `dvc exp save`, we could also make the comparison that it's like `dvc commit` followed by `git commit`. Thoughts?

Reply:
I think it'd be confusing to say that `exp save` includes something like `git commit`, especially so near the top. In fact we've kept all implementation details out of the cmd ref for these so you'd have find a [hidden section in the guide](https://dvc.org/doc/user-guide/experiment-management/experiments-overview#how-does-dvc-track-experiments) to understand what we mean by "like `git commit`" (custom refs).

Reply:
`dvc exp run` can also be used to queue experiments or run them in a temporary directory, so it might be a bit confusing to have this so prominent near the top.
It might no be needed to have this and the `dvc commit`/`git commit` comparison, but we could expand the text later when we talk about using repro  then save, ~line 105.

Reply:
I like having links between closely related commands at the top so all I can think of is change the note to something like this (below), but if you prefer we can move it to the description.

```suggestion
In it's basic usage, this is equivalent to `dvc repro` followed by
`dvc exp save`. See other features below.
```

Reply:
Maybe something even simpler?
```suggestion
When called with no arguments, this is equivalent to `dvc repro` followed by `dvc exp save`.
```